ui/controls.js

```diff
@@ -1717,6 +1740,123 @@ shaka.ui.Controls = class extends shaka.util.FakeEventTarget {
     this.onMouseLeave_();
   }
 
+  /**
+   * @return {shaka.ui.VRManager}
+   */
+  getVR() {
+    goog.asserts.assert(this.vr_ != null, 'Should have a VR manager!');
+    return this.vr_;
+  }
+
+  /**
+   * Returns if a VR is supported.
+   *
+   * @return {boolean}
+   * @export
+   */
+  isPlayingVR() {
+    goog.asserts.assert(this.vr_ != null, 'Should have a VR manager!');
+    return this.vr_.isPlayingVR();
+  }
+
+  /**
+   * Reset VR view.
+   */
+  resetVR() {
+    goog.asserts.assert(this.vr_ != null, 'Should have a VR manager!');
+    this.vr_.reset();
+  }
+
+  /**
+   * Get the angle of the north.
+   *
+   * @return {?number}
+   * @export
```

Comment:
Currently the UI does not provide a UI element with this information (it would be a viewer of the 360 angle that is being observed), that is why the methods are given so that someone can create it. If you think it's not necessary, I agree to remove them.